src/verifact_agents/claim_detector.py

```
@@ -1,97 +1,382 @@
+"""AI-driven claim detection for VeriFact.
+
+This module provides a streamlined claim detection system that uses AI agents
+for intelligent analysis instead of complex rule-based processing.
+"""
+
 import os
+import logging
+import re
+import html
+from typing import List
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)